testsuite/integration/microprofile/pom.xml

```diff
@@ -500,6 +500,7 @@
                     <environmentVariables>
                         <!-- dummy env property for MicroProfile Config testing -->
                         <MPCONFIG_TEST_ENV_VAR>integ-basicmpconfig-test-env-var-value</MPCONFIG_TEST_ENV_VAR>
+                        <TESTCONTAINERS_RYUK_DISABLED>true</TESTCONTAINERS_RYUK_DISABLED>
```

Comment:
I don't think we want this. IIRC Ryuk is used to reap containers on platforms like Linux and disabling it especially on CI will cause problems.

Case in point, the tests fail with something that sounds like it's a direct result of this:

```
Caused by: org.testcontainers.containers.ContainerLaunchException: Timed out waiting for container port to open (localhost ports: [33376, 33377, 33374, 33375] should be listening)
```